<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hexo博客一键部署工具</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>258</width>
     <height>446</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>要上传的文件：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="upload_list"/>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeButton">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>移除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uploadButton">
      <property name="text">
       <string>一键上传</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deployButton">
      <property name="text">
       <string>一键部署</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backupButton">
      <property name="text">
       <string>一键备份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器上文件清单：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="server_file_list"/>
    </item>
    <item>
     <widget class="QPushButton" name="getlistButton">
      <property name="text">
       <string>获取</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downloadButton">
      <property name="text">
       <string>下载</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
